deploy/charts/trust-manager/templates/certificate.yaml

```
@@ -24,6 +24,10 @@ spec:
   dnsNames:
   - "{{ include "trust-manager.name" . }}.{{ include "trust-manager.namespace" . }}.svc"
   secretName: {{ include "trust-manager.name" . }}-tls
+  {{- if and (hasKey .Values.app.webhook.tls "secretTemplate") .Values.app.webhook.tls.secretTemplate }}
+  secretTemplate:
+    {{- toYaml .Values.app.webhook.tls.secretTemplate | nindent 4 }}
+  {{- end }}
```

```suggestion
  {{-  with .Values.app.webhook.tls.secretTemplate }}
  secretTemplate:
    {{- toYaml . | nindent 4 }}
  {{- end }}
```

I think something like this should work. Please verify with "helm template" locally. :-)

---

yes, way better.